frontend/components/bridge/BridgeTransferFlow.tsx

```diff
@@ -37,10 +37,10 @@ const TransferRow = ({ transfer }: { transfer: TokenTransfer }) => {
     <List.Item>
       <Flex justify="space-between" className="w-full">
         <Text>
-          {formatEther(fromAmount)} {fromSymbol}
+          {formatUnitsToNumber(fromAmount, transfer.decimals, 8)} {fromSymbol}
```

Comment:
tiny, could do 
`const { fromAmount, fromSymbol, toSymbol, toAmount, decimals } = transfer;`
in the above and then use 
```suggestion
          {formatUnitsToNumber(fromAmount, decimals, 8)} {fromSymbol}
```